pkgs/by-name/xm/xmountains/package.nix

```diff
@@ -2,36 +2,40 @@
   lib,
   stdenv,
   fetchFromGitHub,
-  xorg,
+  xbitmaps,
+  libX11,
+  imake,
+  gccmakedep,
 }:
 
 stdenv.mkDerivation {
   pname = "xmountains";
-  version = "2.10";
+  version = "2.11-unstable-2024-07-17";
 
   src = fetchFromGitHub {
     owner = "spbooth";
     repo = "xmountains";
-    rev = "aa3bcbfed228adf3fff0fe4295589f13fc194f0b";
-    sha256 = "0dx4n2y736lv04sj41cp1dw8n5zkw5gyd946a6zsiv0k796s9ra9";
+    rev = "5ed87c3b68c91dbae93290182e2c9d247223cae3";
+    hash = "sha256-q2+aJ5ISoSXUW4BaAf9Qq/d+DEBSylceZNKKmN4SbQQ=";
   };
```

Comment:
v2.11 already has the fix, which your `hash` already points at (Note that this comment has the same sha256 hash as your package)
```nix
stdenv.mkDerivation(finalAttrs: {
  pname = "xmountains";
  version = "2.11";

  src = fetchFromGitHub {
    owner = "spbooth";
    repo = "xmountains";
    tag = "v${finalAttrs.version}";
    hash = "sha256-q2+aJ5ISoSXUW4BaAf9Qq/d+DEBSylceZNKKmN4SbQQ=0";
  };
```

Reply:
Well you are right, I think i confused myself somewhere :sweat_smile: